contrib/julia.xml

```diff
@@ -158,7 +158,6 @@
       <item> RegexMatch </item>
       <item> RegexMatchIterator </item>
       <item> RepString </item>
```

Comment:
Could remove `RepString` too while you're at it (probably in other files too; but are they maintained?).

Reply:
Ah, that must have been an oversight in #19867. I don't even know what this file is for.